### PR TITLE
Master to650

### PR DESCRIPTION
**Pull request by sakshi-arora1**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: jsdocs/jsdoc.conf.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
         "includePattern": ".+\\.js(doc)?$",
         "excludePattern": "(^|\\/|\\\\)_",
         "include": ["./../ui.frontend/src",
-            "./../ui.af.apps/src/main/content/jcr_root/apps/core/fd/components",
```
[the remark this answers is not in the document]
Comment (rismehta) [reply jsdocs/jsdoc.conf.json:9]: No, we don't need this for now in master or release/650

[the remark this answers is not in the document]
Comment (sakshi-arora1) [reply jsdocs/jsdoc.conf.json:9]: so we are good.

File: bundles/af-core/pom.xml [hunk 2]
```diff
@@ -484,6 +485,7 @@
         <dependency>
             <groupId>com.adobe.aem</groupId>
             <artifactId>uber-jar</artifactId>
+            <classifier>apis</classifier>
```
Comment (ujjwal5) [inline bundles/af-core/pom.xml:488]: @sakshi-arora1 why have we added this, I see this was added in master/dev about 10 months ago which is not required. The build might fail due to this.

File: examples/core/pom.xml [hunk 3]
```diff
@@ -85,6 +85,7 @@
 <!--                            javax.annotation;version=0.0.0,-->
                             <!-- https://github.com/jhy/jsoup/issues/1616 -->
                             javax.annotation;version=!;resolution:=optional,javax.annotation.meta;version=!;resolution:=optional,
+                            com.adobe.cq.wcm.core.components.models;version="[12.0.0,13.0.0)",
```
Comment (ujjwal5) [inline examples/core/pom.xml:88]: is this required ?

File: bundles/core/pom.xml [hunk 4]
```diff
@@ -489,6 +490,7 @@
         <dependency>
             <groupId>com.adobe.aem</groupId>
             <artifactId>uber-jar</artifactId>
+            <classifier>apis</classifier>
```
Comment (ujjwal5) [inline bundles/core/pom.xml:493]: remove this as well

File: bundles/af-core/pom.xml [hunk 5]
```diff
@@ -89,6 +89,7 @@
 <!--                            javax.annotation;version=0.0.0,-->
                             <!-- https://github.com/jhy/jsoup/issues/1616 -->
                             javax.annotation;version=!;resolution:=optional,javax.annotation.meta;version=!;resolution:=optional,
+                            com.adobe.cq.wcm.core.components.models;version="[12.0.0,13.0.0)",
```
[the remark this answers is not in the document]
Comment (sakshi-arora1) [reply bundles/af-core/pom.xml:92]: model versions of core components will be needed right

File: examples/core/pom.xml (hunk 3 above)
Comment (sakshi-arora1) [reply examples/core/pom.xml:88]: we are consuming wcm models in our WCMUse Pojo classes.

File: bundles/core/pom.xml (hunk 4 above)
Comment (sakshi-arora1) [reply bundles/core/pom.xml:493]: ok

**Pull request merged**